### PR TITLE
Feature/load data of my profile for talents (WIP)

### PR DESCRIPTION
**Pull request by LittleSweetGun**
- Add wallet_address column in users table :
TABLE goodhive.users (
  id SERIAL PRIMARY KEY,
  title VARCHAR(100) NOT NULL,
  job_headline VARCHAR(255) NOT NULL,
  first_name VARCHAR(100) NOT NULL,
  last_name VARCHAR(100) NOT NULL,
  country VARCHAR(100) NOT NULL,
  city VARCHAR(100) NOT NULL,
  phone_country_code VARCHAR(10) NOT NULL,
  phone_number VARCHAR(20) NOT NULL,
  email VARCHAR(255) NOT NULL UNIQUE,
  telegram VARCHAR(255),
  currency VARCHAR(100),
  rate VARCHAR(20),
  about_work TEXT NOT NULL,
  skills VARCHAR(255) NOT NULL,
  image_url VARCHAR (255),
  wallet_address VARCHAR (100) UNIQUE
);

- User has to reconnect to the wallet (or the wallet_address stay empty)
TODO : see why it's happening and how to resolve it

- Can't deploy for now because of error caused by @ts-ignore in auth/nonce and auth/verify

### CODE REVIEW

**Comment by LittleSweetGun** [comment]
#67

File: app/layout.tsx [hunk 1]
```diff
@@ -45,6 +46,7 @@ export default function RootLayout({
   const fetchingStatusRef = useRef(false);
   const verifyingRef = useRef(false);
   const [authStatus, setAuthStatus] = useState<AuthenticationStatus>("loading");
+  const [walletAddress, setWalletAddress] = useState("");
```
Comment (n1c01a5) [inline app/layout.tsx:49]: Can you add the type maybe `string | null`.

File: app/layout.tsx [hunk 2]
```diff
@@ -56,13 +58,7 @@ export default function RootLayout({
...
       try {
         // TODO: check if the user is authenticated
-        const hasCookieNonce = Boolean(false); // Mocking this for now
-
-        if (hasCookieNonce) {
-          setAuthStatus("authenticated");
-        } else {
-          setAuthStatus("unauthenticated");
-        }
+        setAuthStatus("authenticated");
```
Comment (n1c01a5) [inline app/layout.tsx:61]: Why did you remove 

```
if (hasCookieNonce) {
          setAuthStatus("authenticated");
        } else {
          setAuthStatus("unauthenticated");
        }
        ```

File: app/layout.tsx [hunk 3]
```diff
@@ -87,6 +83,7 @@ export default function RootLayout({
       },
 
       createMessage: ({ nonce, address, chainId }) => {
+        setWalletAddress(address);
```
Comment (n1c01a5) [inline app/layout.tsx:86]: We must set the address after the validation of the signature not before !

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

**Comment by n1c01a5** [comment]
@LittleSweetGun ok know we have to put the address to a react context to share it with the others component throw a hook.

File: app/layout.tsx [hunk 4]
```diff
@@ -137,7 +134,7 @@ export default function RootLayout({
   }, []);
 
   return (
-    <html lang="en">
+<html lang="en">
```
Comment (n1c01a5) [inline app/layout.tsx:137]: Bad indentation

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

**Pull request merged**